CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
+cmake_minimum_required (VERSION 3.0)
```

Comment:
uh oh. I will check in which version of cmake it's implemented and try to reduce it. If not I will fix BRs.